src/Microsoft.Health.Dicom.Core/Features/Telemetry/InstanceMeter.cs

```diff
@@ -0,0 +1,43 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Diagnostics.Metrics;
+
+namespace Microsoft.Health.Dicom.Core.Features.Telemetry;
+
+public sealed class InstanceMeter : IDisposable
+{
+    private readonly Meter _meter;
+
+    public InstanceMeter()
+    {
+        _meter = new Meter($"{OpenTelemetryLabels.BaseMeterName}.Instance", "1.0");
```

Comment:
nit: same comment about the `"count" units

Reply:
Done